dask/dataframe/core.py

```diff
@@ -3401,7 +3401,7 @@ def dropna(self):
         return self.map_partitions(M.dropna, enforce_metadata=False)
 
     @derived_from(pd.Series)
-    def between(self, left, right, inclusive=True):
+    def between(self, left, right, inclusive="both"):
```

Comment:
This fixes an unrelated CI failure due to a `FutureWarning` being raised in the new pandas 1.3.0 release (see, for example, [this CI build](https://github.com/dask/dask/runs/2963273504))

```
 E           FutureWarning: Boolean inputs to the `inclusive` argument are deprecated infavour of `both` or `neither`.
```